setup.cfg

```diff
@@ -1,4 +1,11 @@
 [flake8]
 exclude = .git,*migrations*
 max-line-length = 119
-ignore=F401,E711,E302,E201,E501,E303,E231,F841
\ No newline at end of file
+ignore=F401,E711,E302,E201,E501,E303,E231,F841
+
+[black]
+line-length = 200
+target-version = ['py36', 'py37', 'py38']
+skip-string-normalization = true
+include = '\.pyi?$'
+exclude = \.*, \.git, \.mypy_cache, \.venv, blib2to3, tests/data, profiling
```

Comment:
Nitpick: -- can you get rid of the red character at the end by adding a line break?

Reply:
Done